guides/source/routing.md

```diff
@@ -451,7 +466,7 @@ The comments resource here will have the following routes generated for it:
 
 ### Routing Concerns
```

Comment:
I could not comment on the [actual line](https://github.com/bhumi1102/rails/pull/7/files#diff-42e915ace4a21890e4d38f5b6230345d62540977e922f87d04b672db908539e7R504-R508) 

```ruby
namespace :articles do
  concerns :commentable
end
```

maybe it will be useful to expand the equivalence for the namespace example as well?

Reply:
Added words instead, so the code blocks are not repetitive and long.